rest/admin_api.go

```diff
@@ -40,12 +42,14 @@ func (h *handler) handleCreateDB() error {
 	}
 	config.Name = dbName
 
+	validateOIDC := !h.getBoolQuery(paramDisableOIDCValidation)
+
 	if h.server.persistentConfig {
 		if err := config.validatePersistentDbConfig(); err != nil {
 			return base.HTTPErrorf(http.StatusBadRequest, err.Error())
 		}
 
-		if err := config.validate(); err != nil {
+		if err := config.validate(h.rqCtx, validateOIDC); err != nil {
```

Comment:
Think this will be safer if the context has not been previously initialised

```suggestion
		if err := config.validate(h.ctx(), validateOIDC); err != nil {
```